mailparser/mailparser.py

```diff
@@ -226,8 +226,12 @@ def message_id(self):
 
     @property
     def to_(self):
-        return decode_header_part(
-            self._message.get('to', self._message.get('delivered-to')))
+        to_ = self._message.get('to')        
+        if not to_:
+            return None
+        else:
```

Comment:
Here's a header from the Enron dataset that has the issue:

`Message-ID: <9457511.1075857601951.JavaMail.evans@thyme>
Date: Mon, 22 May 2000 00:33:00 -0700 (PDT)
From: john.arnold@enron.com
Subject:
Mime-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Transfer-Encoding: 7bit
X-From: John Arnold
X-To: Jeff Skilling@Enron
X-cc: 
X-bcc: 
X-Folder: \John_Arnold_Dec2000\Notes Folders\'sent mail
X-Origin: Arnold-J
X-FileName: Jarnold.nsf
`